### ISSUE TITLE
Dvorak layout support

### ISSUE THREAD

**Issue by craftyguy** [VladThodo/behe-keyboard#20]
Any plans to add this? Thanks!

**Comment by VladThodo** [comment]
Yeah, multiple layouts will be added, including Dvorak. The development will be slower now, as I started school. 
Thanks for getting in touch!
